blockchain/src/message_pool/pool.rs

```diff
@@ -1,71 +0,0 @@
-// Copyright 2020 ChainSafe Systems
```

Comment:
Might make sense to not touch the message_pool stuff, since @flodesi is working on it. Thoughts?

Reply:
he didn't touch any of this code, which is why I remembered to remove this now. I mean that stuff was loosely ported over from what the spec was when we started (I assume) so probably should be relied on anyway

Reply:
Yeah, I'm not using those files, so it's good to be removed.